File: speech_to_text.py
from rev_ai import apiclient

# create your client
client = apiclient.RevAiAPIClient("02kiXlheeVr0xkYA-oZTevxPeuEWjwVGtyogDlUHyfqmIgFMtR4eX8etW2qyBGTyAoBfjrY6uPBIgZmLuyA4F0nf_g5Hc")
# you can send a local file
job = client.submit_job_local_file("recording.wav")
job_details = client.get_job_details(job.id)
print(repr(job_details), job.id)

# transcript_text = client.get_transcript_text(job.id)
# print(transcript_text)
# client.delete_job(job.id)

File: speech_to_text.js
const RevAiApiClient = require('revai-node-sdk');
 
// Initialize your client with your revai access token
var accessToken = "02kiXlheeVr0xkYA-oZTevxPeuEWjwVGtyogDlUHyfqmIgFMtR4eX8etW2qyBGTyAoBfjrY6uPBIgZmLuyA4F0nf_g5Hc";
console.log(typeof(RevApiClient));
var client = new RevAiApiClient(accessToken);

// you can submit a local file
async function submitFile(fileName)
{
	await client.submitJobLocalFile(fileName);
}

async function getTranscript(id)
{
	await client.getTranscriptText(id);
}
var job = submitFile("recording.wav");
var transcriptText = getTranscript(job.id);
console.log(transcriptText);
client.deleteJob(job.id);
